### PR TITLE
LinFctl(PVCrosscov) = `Covariance`

### PR DESCRIPTION
**Pull request by timweiland**
Use the new `Covariance` object in place of `np.ndarray` / `LinearOperator` to represent covariances.

### CODE REVIEW

**Comment by timweiland** [comment]
Decided to implement the covariance arithmetic after all since I believe it's much more elegant than the alternative. Let me know if you disagree, though

File: src/linpde_gp/randprocs/_gaussian_process/_conditional.py [hunk 1]
```diff
@@ -421,8 +422,9 @@ def _(
 )
 def _(
     self, crosscov: ConditionalGaussianProcess.PriorPredictiveCrossCovariance, /
-) -> pn.linops.LinearOperator:
-    return BlockMatrix([[self(kLa_prev) for kLa_prev in crosscov]])
+) -> Covariance:
+    linop_res = BlockMatrix([[self(kLa_prev).linop for kLa_prev in crosscov]])
+    return LinearOperatorCovariance(linop_res, linop_res.shape[:1], linop_res.shape[1:])
```
Comment (marvinpfoertner) [inline src/linpde_gp/randprocs/_gaussian_process/_conditional.py:427]:
```suggestion
    return LinearOperatorCovariance(linop_res, self.output_shape, crosscov.randvar_shape)
```

**Review by marvinpfoertner — APPROVED** [review]
LGTM! 🙏🏻

**Comment by timweiland** [comment]
:heavy_check_mark: Plots are looking great as always

**Pull request merged**